src/ProxyClient/Invalidation/Ban.php

```diff
@@ -11,13 +11,13 @@
 
 namespace FOS\HttpCache\ProxyClient\Invalidation;
 
-use FOS\HttpCache\ProxyClient\ProxyClientInterface;
+use FOS\HttpCache\ProxyClient\ProxyClient;
 
 /**
  * An HTTP cache that supports invalidation by banning, that is, removing
  * objects from the cache that match a regular expression.
  */
-interface BanInterface extends ProxyClientInterface
+interface Ban extends ProxyClient
```

Comment:
Bannable or Banable is the weirdest of them. Purgeable, Refreshable and Taggable all sound quite reasonable to me.